cpp/src/arrow/compute/kernels/scalar_set_lookup_benchmark.cc

```diff
@@ -91,35 +91,52 @@ static void IsInStringLargeSet(benchmark::State& state) {
 }
 
 static void IndexInInt8SmallSet(benchmark::State& state) {
-  SetLookupBenchmarkNumeric<Int8Type>(state, "index_in_meta_binary", state.range(0));
+  SetLookupBenchmarkNumeric<Int8Type>(state, "index_in_meta_binary", state.range(0),
+                                      1 << 18);
```

Comment:
Hmm, can we avoid the repeated hardcoded value?
You could define a constant such as `constexpr int64 kArrayLengthWithSmallSet = 1 << 18`.

Reply:
done